resources/snakefiles/mapping.smk

```diff
@@ -37,7 +37,7 @@ rule map_reads_bt2:
     input:
         reads = lambda wildcards: expand("output/qc/host_filter/nonhost/{sample}.{read}.fastq.gz",
                                          sample=wildcards.read_sample,
-                                         read=['1','2']),
+                                         read=['R1', 'R2']),
```

Comment:
noice. -- this makes sense.  I think I'm starting to understand the difference of when you should use expand and when to use wildcards.